### PR TITLE
chore(wording): Contributor -> Informer

### PR DESCRIPTION
**Pull request by lutangar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 > _Dis moi_ means _Tell me_ in :fr: French.
 
 **Dismoi** is a web extension that allows anyone to post information directly on any web page you browse.
-If you follow a contributor, his information messages are displayed at the time you visit the pages he has commented on.
+If you follow an informer, his messages are displayed at the time you visit the pages he has commented on.
```
Comment (JalilArfaoui) [inline README.md:8]: It looks like `informant` is the most common term in english :

https://www.linguee.fr/francais-anglais/search?source=auto&query=informateur

Comment (lutangar) [reply README.md:8]: Hum, this is also interesting :
https://wikidiff.com/informer/informant

Comment (JalilArfaoui) [reply README.md:8]: Nice ! Totally convinced !

**Pull request merged**